### PR TITLE
Document /admin

### PR DESCRIPTION
**Pull request by foolip**
I did this because it's so much work to find https://wpt.fyi//admin/results/upload for me every few months that I need it.

### CODE REVIEW

File: docs/admin.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Admin
+
+## Flags
+
+[`/admin/flags`](https://wpt.fyi/admin/flags) allows site-wide settings to be set, similar to the per-user [`/flags`](https://wpt.fyi/flags).
+
+## Flushing caches
+
+[`/admin/cache/flush`](https://wpt.fyi//admin/cache/flush) flushes some caches.
```
Comment (foolip) [inline docs/admin.md]: @Hexcles does just visiting this do something? I didn't try out of fear of side effects.

File: docs/admin.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+## Uploading results
+
+[`/admin/results/upload`](https://wpt.fyi//admin/results/upload) is a form for upload results manually. Please file an issue on this project if you require credentials to upload results manually or via the API.
```
Comment (foolip) [inline docs/admin.md]: @Hexcles what is the API endpoint? I tried to find a link to it but failed.

Comment (foolip) [reply docs/admin.md]: It looks like one has to upload a gzipped report. Should that be documented inline, or here?

File: docs/admin.md (hunk 1 above)
Comment (Hexcles) [reply docs/admin.md]: Yes, which is probably a terrible design...

File: docs/admin.md (hunk 2 above)
Comment (Hexcles) [reply docs/admin.md]: The API is https://github.com/web-platform-tests/wpt.fyi/tree/master/api#results-creation

The file input on the upload UI has `accept=".gz"`.

**Pull request merged**